### DISCUSSION TITLE
guides/density-functions/

### DISCUSSION THREAD

**Discussion by giscus** [misode/misode.github.io#229] [Site]
# guides/density-functions/

https://misode.github.io/guides/density-functions/

**Comment by QWERTY770** [comment]
Where is the "slide" function? It isn't written in the guide, but exists in the source code(1.18.2).

**Reply by JJPAKDG** [reply]
the "slide" function was removed in versions 22w12a and up. The reason I came here was to actually find the closest density function to it so I can replace it.
